### ISSUE TITLE
Training bash scripts for llava-1.6-7B

### ISSUE THREAD

**Issue by GaryGuTC** [TIGER-AI-Lab/VLM2Vec#60]
Greetings~
 
Could you provide the training scripts of llava-1.6-7B to reproduce the results? Or help me check my training script.
I used the script as follows (which copy from the example):
```bash
CUDA_VISIBLE_DEVICES=0,1,2,3,4,5,6,7,8 torchrun --nproc_per_node=8 --master_port=22447 --max_restarts=0 train.py \
  --model_name llava-v1.6-mistral-7b-hf \
  --model_backbone llava_next \
  --output_dir $OUTPUT_DIR  \
  --bf16 --pooling last \
  --lora \
  --dataset_name $dataset_name \
  --subset_name $train_data \
  --num_sample_per_subset 50000 \
  --image_dir $image_dir \
  --max_len 64 --logging_steps 1 \
  --lr_scheduler_type linear --learning_rate 2e-5 --max_steps 2000 \
  --warmup_steps 200 --save_steps 500 --normalize True \
  --temperature 0.02 --per_device_train_batch_size 128 \
  --grad_cache True --gc_q_chunk_size 2 --gc_p_chunk_size 2
```
My loss curve is like this:

![Image](https://github.com/user-attachments/assets/d88a6d32-342f-4ee2-bd44-f02c0f2d90da)

Thanks a lot for your help.

**Comment by memray** [comment]
Hi @GaryGuTC , can you try removing the argument ` --max_len`? It was initially used for phi3 models to truncate text lengths. But for newly-added backbones, it will truncate visual tokens as well. So give it a try and let us know if you still see problems.

Thanks,
Rui
